package/boot/uboot-rockchip/Makefile

```diff
@@ -247,8 +247,10 @@ endef
 
 define U-Boot/rk3588/Default
   BUILD_SUBTARGET:=armv8
-  DEPENDS:=+PACKAGE_u-boot-$(1):trusted-firmware-a-rk3588
-  ATF:=rk3588_bl31_v1.45.elf
+  DEPENDS:= \
+	+PACKAGE_u-boot-$(1):trusted-firmware-a-rk3588 \
```

Comment:
Why does this still depend on `trusted-firmware-a-rk3588`?

Reply:
Because that's the name of the package now built from upstream TF-A.